src/Controller/Security/SecurityController.php

```diff
@@ -9,7 +9,7 @@
  * please view the LICENSE file that was distributed with this source code.
  */
 
-namespace App\Controller;
+namespace App\Controller\Security;
```

Comment:
It's the security for the front, isn't? So i should goes to the `Front` dir?

Reply:
It's used to login users so I don't know, and since the authenticators are in App\Security it looks ok to have it in App\Controller\Security since they are related ?

I'm not sure however.

Reply:
Actually, I don't really understand the new directory structure.
I would split in 3 dir:
* admin
* front
* slack
so if this controller is used to auth use on the frontend, it must go to the front dir

Reply:
I moved from Controller\Security to Controller\Front but I will leave the other ones as they are for now since this is not really what this PR should do.

I will probably open a new PR dedicated to the Controllers howewer, as their naming could also use some improvments :).